### ISSUE TITLE
[Bug Accessibility]: Toggle button is not vocalised correctly

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#567]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-android/issues) for duplicate or closed feature requests

### Your test device

Samsung A53

### OS Version

Android 11

### App version

1.0

### Describe the issue

Go to Components -> Buttons:Icons -> toggle button icon group

-> there is no vocalisation of the button selected 

### Expected Behavior

When Talkback is activated, the state of the button (when it is selected ) should be vocalised
